Controllers/review.controller.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { fetchReview } = require("../Models/reviews.model");
+const app = express();
```

Comment:
No need to invoke the express server here, it only needs exist in app.js. Also you may notice this `app` variable isn't been used in this file at all.